app/views/sites/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= content_for :page_title, "#{@errors.any? ? "Error: " : ""}#{ @site_name_before_update }" %>
```

Comment:
When you first go to the page the title comes out as:
`<title> - Publish teacher training courses - GOV.UK</title>`

It needs "Add a location" at the start. If a user enters content, hits save and there are errors, it should be "Error: Add a location"

Reply:
Done